client/gdrive-picker-helloworld.html

```diff
@@ -0,0 +1,105 @@
+<!--
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!-- [START picker_hello_world] -->
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="utf-8" />
+    <title>Google Picker Example</title>
+
+    <script type="text/javascript">
```

Comment:
JS in html file rather than it's own? I respec it, but not sure what others will say

Reply:
Yea I am also confused about this.

Reply:
Ah, I didn't mean to commit that file. That is just a sample file that I copied from somewhere else.